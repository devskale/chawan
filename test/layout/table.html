<!doctype html>
<style>
* {
	font-family: monospace, monospace
}
table {
	border-spacing: 0.5ch 0
}
</style>
<table style="background-color: blue">
  <caption>
    caption
  </caption>
  <tbody>
  <tr>
    <td style="background-color: red">
      one two three four five six seven eight
    </td>
    <td colspan=2 style="vertical-align: bottom" valign=top>
      internationalization
    </td>
    <td>
      c
    </td>
    <td>
      d
    </td>
    <td valign=bottom>
      lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
      tempor incididunt ut labore et dolore magna aliqua
      いろはにほへとちりぬるをわかよたれそつ one two three
      four five six seven eight nine ten elevenn twelve thirteen fourteen
      sixteen seventeen eighteen nineteen
    </td>
  </tr>
  </tbody>
</table>
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
