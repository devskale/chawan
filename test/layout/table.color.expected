                                    caption
[48;2;0;0;255m [48;2;255;0;0m         [48;2;0;0;255m                           lorem ipsum dolor sit amet, consectetur   [49m
[48;2;0;0;255m [48;2;255;0;0mone two  [48;2;0;0;255m                           adipiscing elit, sed do eiusmod tempor    [49m
[48;2;0;0;255m [48;2;255;0;0mthree    [48;2;0;0;255m                           incididunt ut labore et dolore magna      [49m
[48;2;0;0;255m [48;2;255;0;0mfour five[48;2;0;0;255m                       c d aliqua „ÅÑ„Çç„ÅØ„Å´„Åª„Å∏„Å®„Å°„Çä„Å¨„Çã„Çí„Çè„Åã„Çà„Åü„Çå [49m
[48;2;0;0;255m [48;2;255;0;0msix seven[48;2;0;0;255m                           „Åù„Å§ one two three four five six seven    [49m
[48;2;0;0;255m [48;2;255;0;0meight    [48;2;0;0;255m                           eight nine ten elevenn twelve thirteen    [49m
[48;2;0;0;255m [48;2;255;0;0m         [48;2;0;0;255m                           fourteen sixteen seventeen eighteen       [49m
[48;2;0;0;255m [48;2;255;0;0m         [48;2;0;0;255m internationalization      nineteen                                  [49m
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
